packages/apollo/src/fetch-schema.ts

```diff
@@ -18,47 +20,74 @@ import { EndpointConfig } from "./config";
 import { engineLink, getIdFromKey } from "./engine";
 import { SCHEMA_QUERY } from "./operations/schema";
 
-const introspection = gql(introspectionQuery);
+const introspection = gql(getIntrospectionQuery());
 
-export async function fromFile(
+async function buildIntrospectionSchemaInLocalGraphQLContext(
```

Comment:
I find the naming of this function a bit confusing. Maybe `buildIntrospectionSchemaFromSDL`?

Reply:
Fixed via d31f9a8361f18dee7b8186a6fc8c292c37aa3b7.